packages/terra-overlay/src/Overlay.jsx

```diff
@@ -87,9 +86,17 @@ class Overlay extends React.Component {
     document.addEventListener('keydown', this.shouldHandleESCKeydown);
   }
 
+  componentDidUpdate() {
+    if (this.props.isOpen) {
+      this.disableContainerChildrenFocus();
```

Comment:
We probably need to call  disableContainerChildrenFocus in componentDidMount as well (if isOpen is set)

Reply:
Updated in this commit: https://github.com/cerner/terra-core/pull/2679/commits/c8f96a27a024cf20ad45da9a569d8d713a676128 

I've also updated some of the test examples to verify this behavior where we set the overlay to isOpen="true" on initial mount.